SportsHub/SportsHub.DAL/Data/ApplicationDbContext.cs

```diff
@@ -25,6 +29,10 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             new UserEntityTypeConfiguration().Configure(modelBuilder.Entity<User>());
```

Comment:
Please take a look at `ApplyConfigurationsFromAssembly` method in `ModelBuilder` class.

Reply:
Implemented `ApplyConfigurationsFromAssembly`